include/common/mir/events/scroll_axis.h

```diff
@@ -57,6 +57,12 @@ inline bool operator==(ScrollAxisV1<Tag> const& lhs, ScrollAxisV1<Tag> const & r
            lhs.stop == lhs.stop;
 }
 
+template<typename Tag>
+inline bool operator!=(ScrollAxisV1<Tag> const& lhs, ScrollAxisV1<Tag> const & rhs)
+{
+    return !(lhs != rhs);
+}
+
```

Comment:
> Since C++20, instead of the non-member comparison operators, wouldn't it be better to provide a defaulted member comparison?

Apparently not:

    error: ‘bool mir::events::ScrollAxisV1<Tag>::operator==(const mir::events::ScrollAxisV1<Tag>&) const’ cannot be defaulted